util/projectRequest.ts

```diff
@@ -11,34 +12,102 @@ export async function getProjects(): Promise<Project[]> {
     per_page: 100,
   });
 
-  const filteredData = projectsResponse.filter((repo) => !repo.archived);
+  const projects = mapReposToProjects(projectsResponse);
+  return projects;
+}
+
+//search uclaacm org for good first issues, repos which contain gfis, and match them up
+export async function getGoodFirstIssueProjects(): Promise<GFIProject[]> {
+  const PaginatedOctokit = Octokit.plugin(paginateRest);
+  const octokit = new PaginatedOctokit();
+  const gfiIssuesSearchQuery = 'org:uclaacm+label:"good+first+issue"+is:open';
```

Comment:
This isn't a function constant, so you can move it out of the function; would be cleaner to maintain in the future!